index.html

```diff
@@ -125,6 +132,11 @@ <h2 class="section-header">Us!</h2>
                     -->
                     
                     <!-- Person #6 write your code between here-->
+                    <img src:"https://www.google.com/search?q=google&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiA-piczvjMAhWn6IMKHVPzBjMQ_AUICigE&biw=1280&bih=647#imgrc=d17bslxLqp9bOM%3A"> 
```

Comment:
wrap everything in an `li` element with class `clearfix`, and everything but the image in a div with class `person-content`